internal/config/testdata/nginx-agent.conf

```diff
@@ -15,4 +15,12 @@ data_plane_config:
 client: 
   timeout: 10s
 
+metrics:
+  collector: true
+  otlp_export_url: "localhost:3000"
+  otlp_receiver_url: "localhost:4318"
+  collector_receivers:
+    - hostmetrics
+  collector_config_path: "/var/etc/nginx-agent/nginx-agent-otelcol.yaml"
+
 config_dirs: "/etc/nginx:/usr/local/etc/nginx:/usr/share/nginx/modules:invalid/path"
```

Comment:
maybe we should add ```/var/run/``` to config_dirs and validate that the OTel config is within one of those directories as part of the config load?

Reply:
Great idea, I'll do that!